test/performance/Benchmarks.md

```diff
@@ -0,0 +1,72 @@
+# Benchmarks
+
+Knative performance benchmarks are tests geared towards producing useful
+performance metrics of the knative system. All the raw metrics are stored in
+[mako](https://github.com/google/mako)
+
+## Writing new benchmarks
+
+For creating new benchmarks, follow the steps:
+
+1. Create a new directory under `./test/performance/`.
+2. Create a new [mako config](https://github.com/google/mako/blob/github-push-test-1/docs/GUIDE.md#preparing-your-benchmark).
```

Comment:
With #5125 we will want a prod config and a dev config, and will need to link these into kodata as well.